src/set_resources.py

```diff
@@ -0,0 +1,149 @@
+import socket
+from modify_resources import *
+from remote_execution import *
+
+
+"""
+TEMPLATE FOR SCRIPT:
+Start by declaring the IP by 'IP,<ip_address>'
+Then declare the container id by 'Container,<container_id'
+Then set the desired resources:
+    CPU Throttling (single core) 'CPU,<cpu_percentage>' (1 to 100, 100 being MAX CPU)
+    CPU Core pinning 'CPUC,<num_cores>' Note: Core pinning will override cpu throttling
+    DISK read/write rates (bps) 'DISK,<disk_bps>'
+    NETWORK speeds (bps) 'NET,<net_bps>'
+    *** To reset any of the above resources, set the value to 0
+Continue to declare IP, Containers, and Resources
+**Note: This script is read sequentially, so a container_id will be associated with
+    the most recent declared IP. The same goes with container resources.
+When finished, you must add 'END' to the end.
+
+
+Example Script:
```

Comment:
Not a big deal, but any reason you aren't doing this as a more traditional CSV representation?
example:
container_id, cpu_cores, disk_read, network
xxxxxxxxx, 0, 13,100000
yyyyyyyyy, 1, 14, 100
...